src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +70,58 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    class PostalCode {
+        public String postalCode;
+
+        public PostalCode(String postalCode) {
+            this.postalCode = postalCode;
+        }
+
+        @Override
+        public String toString() {
+            return postalCode;
+        }
+    }
+
+    class Block {
```

Comment:
The separated classes should not be in the same java file with Address class. You need to create a new java file for each class.

Reply:
Also note that the header comment for each class is required.